.vitepress/theme/keycap-palette.css

```
@@ -0,0 +1,223 @@
+ *   Accent:       #7ebab5 (teal)
+ *   Accent dark:  #4a9c97 (for light-mode contrast)
+ *   Primary text: #f6f5f5 (warm white, dark mode)
+ *                 #1a1c1e (dark ink, light mode)
+ *   Dark BG:      #090a0c (deep midnight)
+ *   Light BG:     #f8f9fa (warm paper)
+ *   Slate:        #353a40 (secondary surfaces)
+ *
+ * Contrast: All text/bg combos meet WCAG AA (4.5:1+)
+ * ═══════════════════════════════════════════════════════════════ */
+
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600;700&family=JetBrains+Mono:wght@400;500&display=swap');
```

Stylelint (17.4.0)

[error] Expected "url('https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600;700&family=JetBrains+Mono:wght@400;500&display=swap')" to be "'https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600;700&family=JetBrains+Mono:wght@400;500&display=swap'" (import-notation)

(import-notation)